operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs.md

```diff
@@ -26,6 +26,8 @@ When choosing what to highlight in the screenshot, follow this order:
   * Size your Mattermost Desktop App or browser to capture a screenshot approximately 900px by 640px, excluding browser navigation.
   * Use default 100% Zoom.
   * Tip: Close the right-hand sidebar, and close the team sidebar by leaving all but one team.
+* **Font Size:**
+  * Check that the font size is not too small.
```

Comment:
```suggestion
  * All text should have large enough font size so that text is easily readable.
```

Reply:
^This is not worded properly..but basically the guideline is that if you can't easily read the text, the font size is too small